ape_infura/providers.py

```diff
@@ -24,6 +24,9 @@ def disconnect(self):
     def update_settings(self, new_settings: dict):
         pass
 
+    def chain_id(self):
+        pass
```

Comment:
Need to actually provide the correct functionality
```suggestion
    def chain_id(self) -> int:
        return self,_web3.eth.chain_id
```

Reply:
thank you!